.ci.yaml

```diff
@@ -36,6 +36,20 @@ targets:
           {"dependency": "vs_build"}
         ]
 
+  - name: Windows unit tests master - packages
```

Comment:
Please call this `Windows dart_unit_tests master - packages` so that is consistent with our naming scheme (which tries to be consistent to the extent possible between LUCI and Cirrus, and also across plugins and packages).